src/mqueue.rs

```diff
@@ -64,6 +64,21 @@ impl MqAttr {
     pub const fn flags(&self) -> mq_attr_member_t {
         self.mq_attr.mq_flags
     }
+
+    /// The max number of messages that can be held by the queue
+    pub const fn max_msg(&self) -> mq_attr_member_t {
+        self.mq_attr.mq_maxmsg
+    }
+
+    /// The maximum size of each message (in bytes)
+    pub const fn msg_size(&self) -> mq_attr_member_t {
+        self.mq_attr.mq_msgsize
+    }
+
+    /// The number of messages currently held in the queue
+    pub const fn current_msg(&self) -> mq_attr_member_t {
+        self.mq_attr.mq_curmsgs
+    }
```

Comment:
I'm not sure how I feel about the naming here. `cur` is expanded to `current`, but we don't expand `msg`, and the underscores aren't present in the `libc` definitions.

For consistency with the rest of nix (example below), I'd lean towards using the same names as the underlying fields, minus the `mq_` prefix. @asomers thoughts?

https://github.com/nix-rust/nix/blob/a392647f450f88bf4a162135712ea023e8da3b1a/src/sys/socket/addr.rs#L1270-L1273

Reply:
I'm good with that. Seems more consistent. I can fix it.